### ISSUE TITLE
Series expansion to 1F1

### ISSUE THREAD

**Issue by edwardcao3026** [Nemocas/Nemo.jl#540]
Can anyone advise how to do series expansion to a product of confluent hypergeometric functions 1F1(a, b, x)*1F1(c, d, x) using Nemo? Thanks.

**Comment by wbhart** [comment]
At the moment this is certainly not implemented. Though perhaps someone else knows a way of getting it without a lot of work.

**Comment by fredrik-johansson** [comment]
With a C call, there is acb_hypgeom_pfq_series_direct.

**Comment by fredrik-johansson** [comment]
Though, in this specific case, you might be better off computing the terms of the 1F1 series using recurrence relations.

**Comment by edwardcao3026** [comment]
Do you mind sharing more details about the C call method?

**Comment by wbhart** [comment]
The C function in Arb is documented here:

**Comment by wbhart** [comment]
http://arblib.org/acb_hypgeom.html

But from memory we only wrap acb_poly and do not have an acb_series module in Nemo yet. We only have generic series over acb's, I think. And I think the former would really be required to wrap the C function.

Of course you could hack something together, but I assume you will want to do something with it, not just compute it and look at it.

**Comment by edwardcao3026** [comment]
The link is not displayed...

**Comment by wbhart** [comment]
Yeah sorry, I tried to press SHIFT-ENTER, but pressed some other swizzlestick by mistake. The link is in the next comment.
